specs/protocol/README.md

```diff
@@ -93,42 +106,80 @@ see [here](#signatures) for more details
 ## Resource Kinds
 
 ### `Offering`
-An `Offering` is used by the PFI to describe a currency pair they have to _offer_ including the requirements, conditions, and constraints in order to fulfill that offer.
+An `Offering` is used by the PFI to describe a currency pair they have to _offer_ including the requirements, conditions, and constraints needed fulfill that offer.
```

Comment:
```suggestion
An `Offering` is used by the PFI to describe a currency pair they have to _offer_ including the requirements, conditions, and constraints needed to fulfill the offer.
```

Unclear to me the difference between `requirements` and the other two, `conditions, and constraints...`

Entirely new proposal:

```suggestion
An `Offering` is a resource created by a PFI to define requirements for a given currency pair offered for an exchange.
```

Reply:
i like it!